desktop/apps/gallery_partnerships2/stylesheets/index.styl

```diff
@@ -33,6 +33,12 @@
   @media screen and (max-width: 768px)
     flex-wrap wrap-reverse
 
+.max-width-on-mobile
```

Comment:
I added this helper class in order to set the max-width on some elements on the smaller screens breakpoint. This is related to the first item Daniel mentioned in his QA feedback.